frontend/packages/dev-console/integration-tests/testData/git-import/repos.ts

```diff
@@ -21,6 +22,155 @@ interface GithubRepo {
   folder: string;
 }
 
+const devFileResources: DevfileSuggestedResources = {
```

Comment:
@invincibleJai Should we add this as an additional attribute to the gitImporotRepos list?

```tsx
interface GithubRepo {
  url: string;
  folder: string;
  devFileResources?: DevfileSuggestedResources;
}
```

Reply:
have updated it to load similar to  repo.json, content.json, and package.json.